tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_component.ng.html

```diff
@@ -15,20 +15,16 @@
 limitations under the License.
 -->
 
-<div class="fob" (dblclick)="typeStepRequested()">
-  <span *ngIf="!isTyping"> {{ step | number }} </span>
-  <button
-    aria-label="deselect fob"
-    *ngIf="!isTyping"
-    (click)="removeStep.emit()"
-  >
+<div class="fob">
```

Comment:
Since we've broached the topic of accessibility/markup, probably worth noting again that the fob is not properly marked up as an interactable element. My browser doesn't give me the nice cursor indicating I can drag it. And accessibility readers will have no idea what this is supposed to be.

Reply:
Added cursor pointer and aria-lable. Please let me know if anything is missing. Other things I'm looking at is "aria-grabbed"  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-grabbed 
but that requires event handler and a bit out of scope for this PR.

Reply:
I had to think about this some more. I put some thoughts in https://docs.google.com/document/d/1VGY17H9UkF1pAUzhUuVnfALebFEepKbs-QwFYMtjbh0/edit#heading=h.c4dvmcp8ltnq (Sorry, Google-internal only).

I think we should do the simpler suggestion which requires that we decide the fob is not part of a slider:

1. Make the input tabable/focusable. Allow user to enter text. Mark with aria attributes (role, aria-label might be sufficient?). In which case I think you covered this.

2. The fob is not tabable/focusable. It reacts to changes to the input. It should not be marked with aria attributes. It should use `cursor: grab` and `cursor: grabbing` depending on the state.

Reply:
For now I want to go with (2) as the temporary solution. The downside I can think about is the text clicking and editing is not obvious at all. But from the study it seems unlikely users will edit the fob directly so I leave it for now.